flake/dev/default.nix

```diff
@@ -0,0 +1,49 @@
+{ inputs, ... }:
+{
+  imports = [
+    inputs.treefmt-nix.flakeModule
+    ./dev-shell.nix
+    ./nixpkgs-rev.nix
+    ./packages.nix
+    ./pre-commit.nix
+  ];
+
+  perSystem = {
+    treefmt = {
+      projectRootFile = "flake.nix";
```

Comment:
Treefmt now supports asking `git` where the root of the worktree is, however IIUC treefmt-nix doesn't yet support _not_ configuring the `projectRootFile`...

IIRC `flake.nix` or `flake.lock` is already the default. However, you may want to pick another filepath to indicate the root, as this will false-positive for the dev-partition's flake files.

Reply:
The license could be a good candidate for this